specification/ai/Azure.AI.Unified/models/routes.tsp

```diff
@@ -0,0 +1,110 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./models.tsp";
+import "./chat_completions.tsp";
+import "./embeddings.tsp";
+import "./image_embeddings.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Foundations;
+
+namespace Azure.AI.Unified;
+
+interface Models {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get a deployed model.")
+  @route("/models/{name}")
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {
+      @doc("Name of the model container.")
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_]*$")
+      @maxLength(254)
+      @path
+      name: string;
+    },
+    Model
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("List all deployed models in the project.")
+  @route("/models")
+  @get
+  list is Azure.Core.Foundations.Operation<
+    {
+      @doc("Model publisher to filter models by")
+      @query("$modelPublisher")
+      modelPublisher: string,
+
+      @doc("Model name (the publisher specific name) to filter models by")
+      @query("$modelName")
+      modelName: string,
+
+      @doc("Flag to include models from connections in response. Accepted values are 'true' and 'false'")
+      @query("$includeConnectionModels")
+      includeConnectionModels: string,
+
+      ...StandardListQueryParameters;
+    },
+    Model
+  >;
+
+  @doc("""
+  Gets chat completions for the provided chat messages.
+  Completions support a wide variety of tasks and generate text that continues from or "completes"
+  provided prompt data. The method makes a REST API call to the `/chat/completions` route
+  on the given endpoint.
+  """)
+  @actionSeparator("/")
+  @route("models/chat/completions")
+  op getChatCompletions is Azure.Core.RpcOperation<
```

Comment:
If the /chat/completions, /embeddings and /image/embeddings routes are specified here, we won't be able to use this TypeSpec to emit SDK. The emitted SDK should not have inference in it, as this is already in azure-ai-inference. We need to discuss what to do here.

Reply:
@balapv, can you comment on this?

Reply:
I did not want to add more routes for users to figure out. However, given that we are going towards a consolidated client vs individual packages, does it matter going forward?